caps.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const faker = require('faker');
+
+const Events = require('events');
+
+const events = new Events();
+
+function pickup(payload) {
+  console.log('Event:', payload);
+}
+
+function inTransit(payload) {
+  setTimeout(() => {
+    if(payload.event == 'Pickup') {
+      payload.event = 'In Transit';
+      console.log('Event:', payload);
+    }
+  },1000);
+}
+
+function delivered(payload) {
+  setTimeout(() => {
+    if(payload.event == 'In Transit') {
+      payload.event = 'Delivered';
+      console.log('Event:', payload);
+    }
+  },3000);
+}
```

Comment:
This part will move into driver.js. When you move this code to driver.js you'll have to require event from event.js.